packages/messaging-system/src/components/card/card.jsx

```diff
@@ -0,0 +1,43 @@
+// External dependencies
+import React from 'react';
+import PropTypes from 'prop-types';
+// Internal dependencies
+import styles from './card.module.scss';
+
+export class Card extends React.PureComponent {
+  render() {
+    const { username, lastMessage, profilePicture } = this.props;
+    return (
+      <section className={styles.container}>
+        <div className={styles.chatCard}>
+          <figure>
+            <img
+              src={profilePicture}
+              alt="Profile"
+            />
+          </figure>
+          <ul className={styles.info}>
+            <li className={styles.infoUsername}>{username}</li>
+            <li className={styles.infoMessage}>
+              {lastMessage}
+            </li>
+          </ul>
+        </div>
+      </section>
+    );
+  }
+}
+
+Card.propTypes = {
+  username: PropTypes.string,
+  lastMessage: PropTypes.string,
+  profilePicture: PropTypes.string,
+};
+
+Card.defaultProps = {
```

Comment:
See comment: https://github.com/Platzi-Master-C9/booking-ui/pull/206/files#r887209774

We shouldn't use default values for this, instead we need to make all the props required